manipulator_controller:
  type: position_controllers/JointTrajectoryController
  joints: 
    - shoulder_pan_joint
    - shoulder_lift_joint
    - elbow_joint
    - wrist_1_joint
    - wrist_2_joint
    - wrist_3_joint
  gains:
    shoulder_pan_joint: {p: 100, i: 1, d: 1.0, i_clamp: 1}
    shoulder_lift_joint: {p: 100, i: 1, d: 1.0, i_clamp: 1}
    elbow_joint: {p: 100, i: 1, d: 1.0, i_clamp: 1}
    wrist_1_joint: {p: 100, i: 1, d: 1.0, i_clamp: 1}
    wrist_2_joint: {p: 100, i: 1, d: 1.0, i_clamp: 1}
    wrist_3_joint: {p: 100, i: 1, d: 1.0, i_clamp: 1}

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
